### PR TITLE
2023-09-24 OSSP1 branch

### PR DESCRIPTION
**Pull request by way4data**
Main branch it better than OSSP1 branch.

### CODE REVIEW

**Comment by way4data** [comment]
Discard ossp1 branch

**Comment by way4data** [comment]
Merge two branches~

**Review by way4data — COMMENTED** [review]
main branch is better than ossp1 branch.

**Comment by ksuplee** [comment]
main branch is better than ossp1 branch.

File: screenInOut.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # 사용자로부터부터 이름과 학번을 입력 받기
 name = input("이름을 입력하세요: ")
 student_id = input("학번을 입력하세요: ")
-department = input("학과를 입력하세요: ")
+dept = input("학과를 입력하세요: ")
 
 # Output inputed data
 print("이름은 ", name, "입니다.")
 print("학번은 ", student_id, "입니다.") 
-print("학과는 ", department, "입니다.")
```
Comment (ksuplee) [inline screenInOut.py:9]: Department is clear rather than dept.

**Review by ksuplee — APPROVED** [review]

File: screenInOut.py (hunk 1 above)
Comment (way4data) [reply screenInOut.py:9]: I agree with you.

**Pull request closed without merging**